src/runtime/hvx_128.ll

```diff
@@ -64,6 +66,31 @@ define weak_odr <256 x i8> @halide.hexagon.deinterleave.vb(<256 x i8> %arg) noun
   ret <256 x i8> %r
 }
 
+define weak_odr <128 x i32> @halide.hexagon.deinterleave.vw.dv(<128 x i32> %arg) nounwind uwtable readnone alwaysinline {
```

Comment:
Hmm, may be I was overzealous in implementing this. Let me check. If I really need this.